src/chainstate/burn/operations/peg_in.rs

```diff
@@ -0,0 +1,470 @@
+use stacks_common::codec::StacksMessageCodec;
```

Comment:
Please add the copyright statement that we use in all the other source files to this one.

Reply:
Done.